automation_tools/__init__.py

```diff
@@ -390,6 +390,11 @@ def setup_firewall(definitions=None, flush=True):
         if run('systemctl status firewalld', quiet=True).failed:
             run('systemctl enable firewalld')
             run('systemctl start firewalld')
+        # FIXME attemtp to mitigate "Error: INVALID_ZONE"
+        run('firewall-cmd --list-all-zones', warn_only=True)
```

Comment:
I wanted to be quick... I still wonder if sleep helps... no,I bet ;-)
 Going to refactor